pio-engine/build.sbt

```diff
@@ -0,0 +1,11 @@
+import AssemblyKeys._
+
+assemblySettings
+
+name := "predictionio-load-forecasting"
+
+libraryDependencies ++= Seq(
+  "org.apache.predictionio"    %% "apache-predictionio-core" % "0.10.0-incubating" % "provided",
+  "org.apache.spark"  %% "spark-core"   % "1.3.0"           % "provided",
+  "org.apache.spark"  %% "spark-mllib"  % "1.3.0"           % "provided"
```

Comment:
Should we use 1.5.2 as we are using Spark 1.5.2?
[here](https://mvnrepository.com/artifact/org.apache.spark/spark-core_2.10/1.5.2.2.3.4.51-1) is the latest release for 1.5.2.x